i18n/translations/en/common.json

```diff
@@ -221,16 +221,16 @@
   "number": "Number",
   "saveAndResume": {
     "saveBtn": "Save to device",
-    "resumeBtn": "Resume",
-    "formRestored": "Form values restored from saved file.",
+    "resumeBtn": "Resume form",
+    "formRestored": "Your form answers so far were successfully uploaded.",
```

Comment:
Notes:

This is for the toast where we let the user know they we've re-loaded the form values.  

Once the feature flag is gone this will also show for a session re-load.